### PR TITLE
Adds screenshot tests to ComposeStarter

### PR DESCRIPTION
**Pull request by garanj**
Adds screenshot tests to `ComposeStarter`

Uses [Roborazzi](https://github.com/takahirom/roborazzi) to create screenshots for common device sizes.

### CODE REVIEW

File: ComposeStarter/app/src/test/java/presentation/WearScreenshotTest.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+    abstract val device: WearDevice
+
+    fun runTest(content: @Composable () -> Unit) {
+        RuntimeEnvironment.setQualifiers("+w${device.dp}dp-h${device.dp}dp")
+        composeRule.setContent {
+            content()
+        }
+
+        composeRule.onRoot().captureRoboImage(
```
Comment (yschimke) [inline ComposeStarter/app/src/test/java/presentation/WearScreenshotTest.kt:44]: Can you pass in 

```
roborazziOptions = RoborazziOptions(
            recordOptions = RoborazziOptions.RecordOptions(
                applyDeviceCrop = true
            )
        )
```

**Review by yschimke — APPROVED** [review]

File: ComposeStarter/app/src/test/java/presentation/WearScreenshotTest.kt (hunk 1 above)
Comment (garanj) [reply ComposeStarter/app/src/test/java/presentation/WearScreenshotTest.kt:44]: Ah that's awesome, that's what I was looking for!!

File: ComposeStarter/app/src/test/java/presentation/WearScreenshotTest.kt [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    abstract val device: WearDevice
+
+    @OptIn(ExperimentalRoborazziApi::class)
+    fun runTest(content: @Composable () -> Unit) {
+        RuntimeEnvironment.setQualifiers("+w${device.dp}dp-h${device.dp}dp")
+        composeRule.setContent {
+            content()
+        }
+
+        composeRule.onRoot().captureRoboImage(
```
Comment (yschimke) [inline ComposeStarter/app/src/test/java/presentation/WearScreenshotTest.kt:47]: You might also want to set 

```
            compareOptions = RoborazziOptions.CompareOptions(
                resultValidator = ThresholdValidator(tolerance)
            )
```

And make it overridable in each specific test.

File: ComposeStarter/app/src/test/screenshots/GreetingScreenTest_pixelwatch.png
Comment (yschimke) [inline ComposeStarter/app/src/test/screenshots/GreetingScreenTest_pixelwatch.png:1]: why square?

File: ComposeStarter/app/src/test/java/presentation/GreetingScreenTest.kt [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.ParameterizedRobolectricTestRunner
+
+@RunWith(ParameterizedRobolectricTestRunner::class)
+class GreetingScreenTest(override val device: WearDevice) : WearScreenshotTest() {
+
+    @Test
+    fun greetingScreenTest() = runTest {
+        GreetingScreen(greetingName = "screenshot", onShowList = {})
```
Comment (yschimke) [inline ComposeStarter/app/src/test/java/presentation/GreetingScreenTest.kt:28]: Can we wrap with 

```
AppScaffold {}
```

So we get TimeText and PositionIndicator?

File: ComposeStarter/app/src/test/java/presentation/WearScreenshotTest.kt (hunk 2 above)
Comment (garanj) [reply ComposeStarter/app/src/test/java/presentation/WearScreenshotTest.kt:51]: Done!

File: ComposeStarter/app/src/test/screenshots/GreetingScreenTest_pixelwatch.png
Comment (garanj) [reply ComposeStarter/app/src/test/screenshots/GreetingScreenTest_pixelwatch.png:1]: Yeah, spotted that, removed! Artefact of an old name...

File: ComposeStarter/app/src/test/java/presentation/GreetingScreenTest.kt (hunk 3 above)
Comment (garanj) [reply ComposeStarter/app/src/test/java/presentation/GreetingScreenTest.kt]: Done!

**Pull request merged**